sources/chargen.js

```diff
@@ -684,20 +725,27 @@ $(document).ready(function() {
         const previewToDraw = {};
         const animation =  $(this).data(`layer_1_custom_animation`);
 
-        if($(this).data(`layer_1_${getBodyTypeName()}`) === undefined){
-          previewToDraw.link = $(this).data(`layer_1_${getBodyTypeName()}`);
+        if($(this).data(`layer_1_${getBodyTypeName()}`) === undefined) {
+          var imageLink = $(this).data(`layer_1_${getBodyTypeName()}`);
```

Comment:
Let's (pun intended) avoid var in new code. Use let.

Reply:
Fixed it (see: https://github.com/LiberatedPixelCup/Universal-LPC-Spritesheet-Character-Generator/pull/71/commits/a4459de5ab849becfecb87bb67d92423516a3872)